src/Listener/RelatedModelsListener.php

```diff
@@ -51,7 +52,8 @@ public function publishRelatedModels($action = null)
                 continue;
             }
 
-            $query = $association->find()->find('list');
+            $finder = $this->_finder($association);
```

Comment:
Undefined method - was this intended to be protected, or public like in the tests?